### DISCUSSION TITLE
Can I join in your team sir?

### DISCUSSION THREAD

**Discussion by dmb-kirya** [ROM-builders/temporary#17937] [Q&A]
Hello sir, I have successfully built ROM with PR, can I ask you add me in your team?

My DT:
Device tree - https://github.com/dmb-kirya/device_xiaomi_alioth
Device-common tree - https://github.com/dmb-kirya/device_xiaomi_sm8250-common
Kernel - https://github.com/dmb-kirya/nexus_kernel_xiaomi_sm8250
Vendor - https://github.com/dmb-kirya/vendor_xiaomi_alioth
Vendor-common - https://github.com/dmb-kirya/vendor_xiaomi_sm8250-common

My successfull build:
https://cirrus-ci.com/build/6407868237217792

My telegram:
https://t.me/kiryash1k

Thank you!

**Comment by Apon77** [comment]
need some time and follow #learn from telegram

**Reply by dmb-kirya** [reply]
Thank you for answering sir!
